frontend/app/src/components/form/inputs/TextAreaInput.tsx

```diff
@@ -97,7 +97,7 @@ export default function TextAreaInput({
         }}
         className={active ? 'euiFormRow_active' : (value ?? '') === '' ? '' : 'euiFormRow_filed'}
         border={borderType}
-        label={labeltext}
+        label={labeltext === 'Description' ? `${labeltext} *` : labeltext}
```

Comment:
we should try to keep logic that is not related to the common component out of here